Soccar [Game-Server]/src/nl/soccar/gameserver/model/command/CommandListener.java

```diff
@@ -7,24 +7,53 @@
 import nl.soccar.library.enumeration.Privilege;
 
 /**
+ * Listener for listening to Commands.
+ *
  * @author PTS34A
  */
 public abstract class CommandListener {
 
     private final Privilege minimumPrivilege;
     private final boolean hostOnly;
 
+    /**
+     * Initializes the CommandListener class.
+     *
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)